src/tests/ftest/util/apricot/apricot/test.py

```diff
@@ -196,6 +198,24 @@ def setUp(self):
                     "Test requires {} {}; {} specified".format(
                         expected_count, host_type, actual_count))
 
+    def spdk_setup(self):
+        '''SPDK setup for NVMe drives'''
+        try:
+            if self.nvme_parameter == "nvme":
+                spdk.nvme_setup(self.hostlist_servers)
+        except spdk.SpdkFailed as error:
+            self.fail("Error setting up NVMe: {}".format(error))
+
+    def spdk_cleanup(self):
+        '''SPDK cleanup for NVMe drives'''
+        if self.nvme_parameter == "nvme":
+            spdk.nvme_cleanup(self.hostlist_servers)
+
+    def setUp(self):
+        """Set up each test case."""
+        super(TestWithServers, self).setUp()
+        self.init_server_param()
+        self.spdk_setup()
```

Comment:
AlI don't think that we should expose all the SPDK stuff there. I would advise to integrate all of this into the server utils and just called storage prep-nvme automatically in the server_utils package.

Reply:
Will remove SPDK and implement function in server_utils.